opencsp/app/sofast/SofastService.py

```diff
@@ -263,14 +315,21 @@ def get_exposure(self) -> float | None:
         return self.image_acquisition.exposure_time
 
     def set_exposure(self, new_exp: float) -> None:
-        """Sets camera exposure time value to the given value"""
+        """Sets camera exposure time value to the given value (seconds)"""
```

Comment:
This actually varies by camera. For instance, with the Basler cameras, it is micro seconds.

Reply:
Yeah, that makes sense.

What do you think about using microseconds as a standard unit of time for exposure, and doing the conversion in the image_acquisition class? (see commit https://github.com/sandialabs/OpenCSP/commit/2e74cefb3a824ef97476fe85c6888e860dfd1fea)

Reply:
Ooh, I like that. Good idea!